### PR TITLE
add support for the beaglebone-black

### PR DESCRIPTION
**Pull request by chombourger**
I haven't tried booting the image just yet

### CODE REVIEW

**Comment by chombourger** [comment]
boot and smoke tested. this PR may now be reviewed

File: meta-isar/recipes-bsp/u-boot/u-boot-beaglebone-black_2021.10.bb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# ---------------------------------------------------------------------------
```
Comment (vj-kumar) [inline meta-isar/recipes-bsp/u-boot/u-boot-beaglebone-black_2021.10.bb:1]: Can't we use the MLO & u-boot.img in /usr/lib/u-boot/am335x_boneblack/ via u-boot-omap package and avoid this recipe?

https://packages.debian.org/bullseye/armhf/u-boot-omap/filelist

Comment (chombourger) [reply meta-isar/recipes-bsp/u-boot/u-boot-beaglebone-black_2021.10.bb:1]: yes, this version appears to be compatible. did the change locally and the system was booted ok. doing a clean build to double-check

Comment (chombourger) [reply meta-isar/recipes-bsp/u-boot/u-boot-beaglebone-black_2021.10.bb]: build & boot tested clean build: all ok

**Review by vj-kumar — APPROVED** [review]

**Pull request merged**